### DISCUSSION TITLE
My car is buick enclave,same platform with xt6 and blazer. how to support my car?

### DISCUSSION THREAD

**Discussion by xyuelin** [opgm/openpilot#103] [Q&A]
My car is buick enclave,same platform with xt6 and blazer. how to support my car?

**Comment by RSZR2** [comment]
I have a Blazer with ACC and LDW that I have been able to fingerprint but can't get past there.

**Reply by nworb-cire** [reply]
Hi @RSZR2, probably better to reach out on the discord server linked in the repo readme.
